src/styles/main.scss

```diff
@@ -1,3 +1,83 @@
-body {
-  margin: 0;
+$gap: 1px;
+$columnCount: 7;
+$cellSize: 100px;
+$padding: 10px;
+$calendarWidth: ($columnCount * $cellSize) + (($columnCount - 1)
+* $gap) + $padding;
+
+$monthDays: (
+  "sun": 0,
+  "mon": 1,
+  "tue": 2,
+  "wed": 3,
+  "thu": 4,
+  "fri": 5,
+  "sat": 6,
+);
+
+$monthLengths: 28, 29, 30, 31;
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  max-width: $calendarWidth;
+  margin: 0 auto;
+  padding: $padding;
+  gap: $gap;
+
+  &__day {
```

Comment:
we can also use flex to ceneter `__day` - then there will be no point in styling `::before`